enterprise/coderd/coderd.go

```diff
@@ -377,7 +377,7 @@ func New(ctx context.Context, options *Options) (_ *API, err error) {
 		//
 		// We may in future decide to scope provisioner daemons to organizations, so we'll keep the API
 		// route as is.
-		r.Route("/organizations/{organization}/provisionerdaemons", func(r chi.Router) {
+		r.Route("/organizations/{organization}/provisionerdaemons/serve", func(r chi.Router) {
```

Comment:
nit: does this change give any special benefits? Most likely I'm missing something...

Reply:
We removed the other endpoints from here that were under `/organizations/{organization}/provisionerdaemons` and only `/organizations/{organization}/provisionerdaemons/serve` remains. There's no reason to keep the broader one and IIRC it also interfered with the AGPL endpoint.